src/main/java/com/stripe/model/Review.java

```diff
@@ -70,7 +70,7 @@
   @SerializedName("open")
   Boolean open;
 
-  /** The reason the review was opened. One of `rule` or `manual`. */
+  /** The reason the review was opened. One of `rule`, `manual`, or `issuer_fraud_record`. */
```

Comment:
technically the last one is supposed to be secret and should not be available publicly.

Reply:
bump